api_server/common/constants/auth/email_lambda_client.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class EmailLambdaClientConstants(enum.Enum):
+    """EmailConfirmationLambdaClient constants."""
+    WAIT_1_SECOND = 1
```

Comment:
I don't know know what about enums, but I think it's better to name constants not as verbs (entity which makes some action), but name like something which describes what's in this constant, e.g. `ONE_SECOND`, `SECOND_WAITING`

Reply:
Changed in new commit.